### PR TITLE
Content designer adding/leaving process for 2nd Line

### PR DESCRIPTION
**Pull request by kevindew**
Trello: https://trello.com/c/m1f54bYg/183-define-process-for-content-designers-to-work-on-smart-answers

This outlines the reasons why a content designer may have access to
GitHub and the processes that someone on 2nd Line should follow to
grant/remove their access.

Note to self:

- [ ] After merge update [wiki](https://gov-uk.atlassian.net/wiki/spaces/GOVUK/pages/515440654/Adding+and+removing+users+to+and+from+the+GOV.UK+content+designers+GitHub+team)

### CODE REVIEW

File: source/manual/content-designer-github-access.html.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+When a content designer joins they will need to be added to our monitoring
+software and then can be added to the GitHub team. These are the steps to
+follow:
+
+1. Edit the [users.yml][] file in [govuk-user-reviewer][] with the following
+   details for each designer: their name, GitHub username and role in
+   GOV.UK - [example][user-example]
+1. Raise this as a pull request
+1. Once PR is merged email gds-github-owners@digital.cabinet-office.gov.uk
```
Comment (tijmenb) [inline source/manual/content-designer-github-access.html.md]: Can we change this to govuk-senior-tech-members@digital.cabinet-office.gov.uk? The GDS GitHub owners mailing list goes out to a lot more people, who won't have context. I wouldn't want to receive emails for access requests in other parts of GDS.

Comment (bevanloon) [reply source/manual/content-designer-github-access.html.md]: @tijmenb 👍  fixed in 212475f

**Comment by kevindew** [comment]
@tijmenb I've got those changes in there for you - and reworked the text slightly so it's less 2nd line focussed (I expect mostly that people on 2nd line will be asked to do this - but do agree that this doesn't seem 2nd line specific)

Also with the change to email address, the gds-github-owners was consistent with our leaver process - I've now updated the leaver ticket to also use the govuk-senior-tech-members address https://trello.com/c/IQIV54Pc/72-leaver-template-v20 FYI

**Comment by tijmenb** [comment]
@kevindew thank you!

**Pull request merged**